conans/model/conan_file.py

```diff
@@ -165,10 +165,6 @@ def package_folder(self):
     def generators_folder(self):
         return self.folders.generators_folder
 
-    @property
-    def imports_folder(self):
```

Comment:
I think this might be a bit painful for 1.X -> 2 migrations, but lets move forward and we will see.

Reply:
I think, migrating the copy tool and with generating it shouldn't be that hard.